contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.3.0--3.4.0.sql

```diff
@@ -657,5 +657,1012 @@ $BODY$
 LANGUAGE plpgsql
 IMMUTABLE;
 
+SET allow_system_table_mods = on;
+
+ALTER TABLE sys.babelfish_sysdatabases ADD COLUMN IF NOT EXISTS orig_name TEXT COLLATE "C";
+
+UPDATE sys.babelfish_sysdatabases SET orig_name = name WHERE orig_name IS NULL;
+
+ALTER TABLE sys.babelfish_sysdatabases ALTER COLUMN orig_name SET NOT NULL;
```

Comment:
Now this kind of changes make multi-version dump/restore complicated since we are adding a new column and expect it to be NOT NULL (unfortunately we have done this in past). Is it possible to skip this and not make this column NOT NULL. As a result we should be able to write the logic in system views so that if `orig_name` is NULL than take it from `dbname`?